backend/doc/apache.md

```diff
@@ -41,6 +41,7 @@ Create a file named `/etc/apache2/sites-available/tracim.conf` containing:
         ProxyPass /ui http://127.0.0.1:6543/ui
         ProxyPassReverse /ui http://127.0.0.1:6543/ui
 
+        ProxyPassMatch /api/(v2/users/.*/live_messages)$ http://127.0.0.1:7999/api/$1
```

Comment:
Does proxyPassMatch does same as ProxyPass + ProxyPassReverse ?

Reply:
note: should be updated later (/v2 will be removed)

Reply:
> Does proxyPassMatch does same as ProxyPass + ProxyPassReverse ?

It is not clear if the ProxyPassReverse directive works with regular expressions (and there is no ProxyPassMatchReverse): http://httpd.apache.org/docs/2.2/mod/mod_proxy.html#proxypassreverse

I just found an [article](https://botproxy.net/docs/how-to/how-to-write-regex-for-apache-proxypassmatch-to-reverse-proxy-api-calls/) indicating that it would work so we can try. I'll try.

Reply:
Ok, please notify if there is some change to be done, we do need to update tracim apache documentation too:
https://github.com/tracim/tracim/pull/3137